### PR TITLE
fixes #34 creates separate renameFiles method for sonarr and radarr

### LINKED ISSUE #34: Sonarr rename command now requires series_id
```python
self.sonarr_cli._sendCommand(
    {
        "name": "RenameFiles",
        "files": [1,2,3],
        "seriesId": 1,
    }
)
```

### PR DESCRIPTION
**Pull request by hollanbm**

### CODE REVIEW

File: src/pycliarr/api/sonarr.py [hunk 1]
```diff
@@ -336,4 +335,23 @@ def get_queue(
         """
         return self._get_queue(
             page, sort_key, page_size, sort_dir, options={"includeUnknownSeriesItems": include_unknown}
```
Comment (vche) [inline src/pycliarr/api/sonarr.py:337]: you moved the parenthesis, so this is not closed anymore, it won't work

File: src/pycliarr/api/sonarr.py [hunk 2]
```diff
@@ -336,4 +335,23 @@ def get_queue(
...
+
+
+    def rename_files(
+        self, file_ids: List[int], serie_id: int
+    ) -> Dict[str, Any] | List[Dict[str, Any]]:
```
Comment (vche) [inline src/pycliarr/api/sonarr.py]: this is not valid python typing. you should use union. i've created the `json_data` type for this.

**Review by vche — CHANGES_REQUESTED** [review]

**Comment by vche** [comment]
I've left a few comments, this code cannot currently work, a few small changes. Here are the changes you need to do for your PR to work.

[fix_pr_35.patch](https://github.com/vche/pycliarr/files/15239913/fix_pr_35.patch)

File: src/pycliarr/api/sonarr.py (hunk 1 above)
Comment (hollanbm) [reply src/pycliarr/api/sonarr.py:337]: Fixed with your patch, thanks!

File: src/pycliarr/api/sonarr.py (hunk 2 above)
Comment (hollanbm) [reply src/pycliarr/api/sonarr.py]: fixed with your patch, thanks!

**Review by vche — APPROVED** [review]

**Pull request merged**